README.md

```diff
@@ -62,27 +64,26 @@ If you use Terraform to manage your infrastructure, MongoDB for VS Code helps yo
 
 ## Extension Settings
 
-- `mdb.shell`: The MongoDB shell to use (The new `mongosh` or the legacy `mongo`).
-- `mdb.showMongoDBConnectionExplorer`: Show or hide the MongoDB Connection explorer.
-- `mdb.showMongoDBPlaygrounds`: Show or hide the MongoDB Playgrounds explorer.
-- `mdb.showMongoDBHelpExplorer`: Show or hide the MongoDB Help explorer.
-- `mdb.defaultLimit`: The number of documents to fetch when viewing documents from a collection.
-- `mdb.confirmRunAll`: Show a confirmation message before running commands in a playground.
-- `mdb.confirmDeleteDocument`: Show a confirmation message before deleting a document in the tree view.
-- `mdb.excludeFromPlaygroundsSearch`: Exclude files and folders while searching for playground in the the current workspace.
-- `mdb.connectionSaving.hideOptionToChooseWhereToSaveNewConnections`: When a connection is added, a prompt is shown that let's the user decide where the new connection should be saved. When this setting is checked, the prompt is not shown and the default connection saving location setting is used.
-- `mdb.connectionSaving.defaultConnectionSavingLocation`: When the setting that hides the option to choose where to save new connections is checked, this setting sets if and where new connections are saved.
-- `mdb.useDefaultTemplateForPlayground`: Choose whether to use the default template for playground files or to start with an empty playground editor.
-- `mdb.uniqueObjectIdPerCursor`: The default behavior is to generate a single ObjectId and insert it on all cursors. Set to true to generate a unique ObjectId per cursor instead.
-- `mdb.sendTelemetry`: Opt-in and opt-out for diagnostic and telemetry collection.
-
-![Settings](resources/screenshots/settings.png)
-
+| Setting                                                            	| Description                                                                                                                                                                                                                          	| Default                                                                                                                    	|
```

Comment:
Maybe we could reformat the table to optimize the columns' width. The text is squished in the middle row.

Reply:
Column width is not easy to control in markdown tables. Let me see how it looks if I add a new line for the longer settings.